### PR TITLE
Add parsers for audit

### PR DESCRIPTION
**Pull request by kaligrafy**
## Summary by CodeRabbit

- **New Features**
  - Automatic parsing/normalization for interviews, homes, trips and visited places; runtime parser mapping; added public config properties: countryCode, startDate, endDate, and mapAerialTilesUrl.

- **Tests**
  - Comprehensive unit and integration tests covering each parser, wiring, edge cases, concurrency, large-data and idempotence.

- **Documentation**
  - Coding rules updated: doc line prefix changed to -, prefer parametric tests, 4-space indentation, and referenced formatter configs.

- **Chores**
  - Parsers wired into server runtime/configuration.

### CODE REVIEW

File: survey/src/admin/parsers/visitedPlace.parser.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        return visitedPlaceAttributes;
+    }
+
+    // update start and end times from arrival/departure times:
+    if (visitedPlaceAttributes.arrivalTime !== undefined) {
+        visitedPlaceAttributes.startTime = Number(visitedPlaceAttributes.arrivalTime);
+    }
+    if (visitedPlaceAttributes.departureTime !== undefined) {
+        visitedPlaceAttributes.endTime = Number(visitedPlaceAttributes.departureTime);
+    }
```
[the remark this answers is not in the document]
Comment (kaligrafy) [reply survey/src/admin/parsers/visitedPlace.parser.ts]: you asked me to put them there before... I think it is ok to keep them :-)

File: survey/src/admin/parsers/__tests__/visitedPlace.parser.test.ts [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+    });
+
+    it('should handle null corrected_response gracefully', () => {
+        const visitedPlaceAttributes: ExtendedVisitedPlaceAttributes = {
+            _uuid: 'test-visited-place-uuid',
+            arrivalTime: 28800,
+            departureTime: 32400
+        };
+
+        const correctedResponse =null as any;
```
[the remark this answers is not in the document]
Comment (kaligrafy) [reply survey/src/admin/parsers/__tests__/visitedPlace.parser.test.ts]: fixed

**Comment by tahini** [comment]
Par exemple pour `segments`, qui utilise déjà plusieurs widgets builtin dans Evolution, s'il y a du traitement à faire avec les champs builtin, il faudrait que ça se fasse dans Evolution directement.

**Comment by kaligrafy** [comment]
https://github.com/chairemobilite/evolution/issues/1284
